resources/monitoring/charts/prometheus-istio/values.yaml

```diff
@@ -384,8 +380,6 @@ configmapReload:
     ## configmap-reload container image
     ##
     image:
-      repository: jimmidyson/configmap-reload
-      tag: v0.4.0
```

Comment:
we are not using that image/functionality, please just comment out and do not specify in the listing. Please revert related changes of defining the image to have it more easy to update the chart

Reply:
This change was reverted